aspnetcore/performance/caching/memory/sample/WebCache/project.json

```diff
@@ -0,0 +1,49 @@
+﻿{
+  "dependencies": {
+    "Microsoft.AspNetCore.Diagnostics": "1.2.0-preview1-22808",
```

Comment:
Yikes... mix of LTS (1.0.x) FTS (1.1.x) and dev (1.2.x) packages. Pick one of them not all of them.